app/presenters/teaching_events/event_presenter.rb

```diff
@@ -95,5 +107,11 @@ def show_provider_information?
     def show_venue_information?
       !@event.is_virtual && @event.building.present?
```

Comment:
Do online events have a building? it may be worth checking `is_online` rather than `is_virtual`, as all virtual events should be online, but not all online events are virtual